stable/locust/README.md

```diff
@@ -21,20 +21,24 @@ This chart will do the following:
 To install the chart with the release name `locust-nymph` in the default namespace:
 
 ```bash
-helm install -n locust-nymph --set master.config.target-host=http://site.example.com stable/locust
+helm install -n locust-nymph --set master.config.target-url=http://site.example.com stable/locust
 ```
 
 | Parameter                    | Description                             | Default                                               |
 | ---------------------------- | ----------------------------------      | ----------------------------------------------------- |
 | `Name`                       | Locust master name                      | `locust`                                              |
+| 'createTestScripts'          | Use Helm to create the test scripts     | 'true'                                                |
 | `image.repository`           | Locust container image name             | `quay.io/honestbee/locust`                            |
 | `image.tag`                  | Locust Container image tag              | `0.7.5`                                               |
```

Comment:
update the image tag here as well

Reply:
Updated.  This will change once Locust.io setups their own docker hub repo.  I can update it then.  I am following up with them on this here: https://github.com/locustio/locust/pull/850#issuecomment-424425247